docs/version0.41.md

```diff
@@ -44,9 +45,15 @@ A VM maintains multiple platform threads that are used as carrier threads to run
 
 For more information, see [`-XX:[+|-]ShowCarrierFrames`](xxshowcarrierframes.md). ![End of content that applies to Java 21 (LTS) and later](cr/java_close_lts.png)
 
+### ![Start of content that applies to Java 21 (LTS) and later](cr/java21plus.png) New `-XX:ContinuationCache` option added
+
+When a virtual thread is created, it is associated with a Continuation object. A virtual thread saves its current state in its continuation object and later resumes from that state. When the continuation implementation is complete, instead of freeing the allocated memory, the VM stores such a memory in a continuation cache. To create a new continuation object, the VM can directly use the cached structure.
+
+You can optimize the virtual thread performance by increasing the Continuation tier 1 and 2 cache size with the [`-XX:ContinuationCache`](xxcontinuationcache.md) option. ![End of content that applies to Java 21 (LTS) and later](cr/java_close_lts.png)
```

Comment:
I would prefer to use "tuning" instead of "increasing" here, depending on the application, increasing cache size may not always improve performance.